GETTING_STARTED.md

```diff
@@ -0,0 +1,121 @@
+Using this repo:
+
+This repo can be installed via cargo
+
+    cargo install clvm_tools_rs
+    
+The most current version of the language is in the rollup branch:
+
+    20221005-rollup
```

Comment:
I can pretty much guarantee this will fall out of date quickly. I don't think this is a wise place to put this information.

Reply:
perhaps, but since it's for people at chia, this is where everything is integrated (in case some questions are of the form "i tried assign or lambda and they didn't work").

given the pace of reviews, i'm uncertain how quickly that information will actually become stale.